### DISCUSSION TITLE
Is nonempty deprecated?

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [colinhacks/zod#2853] [Q&A] [answered]
![image](https://github.com/colinhacks/zod/assets/10191085/96c7d6cc-1e78-4f38-9641-3063f4100117)

![image](https://github.com/colinhacks/zod/assets/10191085/b97b4092-71a2-49ac-b276-d9e7944a3a0b)

**Comment by JacobWeisenburger** [comment, marked as answer]
Yes
